<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameScreenWidget</class>
 <widget class="QWidget" name="GameScreenWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Game{
background-image: url(:/game/Grafiki/game_bg.png);
}
QPushButton
{
font-size: 18px;
}
QLabel
{
font-size: 15px;
}</string>
  </property>
  <widget class="QFrame" name="Game">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>720</width>
     <height>540</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="CardsWidget" name="CurrentPlayer" native="true">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>400</y>
      <width>581</width>
      <height>185</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="OpponentCardsWidget" name="Player4" native="true">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>170</y>
      <width>120</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="TableCard" native="true">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>180</y>
      <width>121</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="OpponentCardsWidget" name="Player2" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>120</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="OpponentCardsWidget" name="Player3" native="true">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>301</width>
      <height>170</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="cardButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">CardsWidget{
	background-image:&quot;&quot;;
}</string>
    </property>
    <property name="text">
     <string>Ciągnij kartę</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">CardsWidget{
	background-image:&quot;&quot;;
}</string>
    </property>
    <property name="text">
     <string>Wyjście</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCurrentPlayer">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>375</y>
      <width>581</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Player</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OpponentCardsWidget</class>
   <extends>QWidget</extends>
   <header>opponentcardswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CardsWidget</class>
   <extends>QWidget</extends>
   <header>cardswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
